.travis.yml

```diff
@@ -42,7 +35,6 @@ jobs:
     - stage: test
```

Comment:
Remove this job entirely, which is this item in the list, with its stage and env.

Reply:
ok